lib/floki/html_parser/fast_html.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Floki.HTMLParser.FastHtml do
+  @moduledoc false
+
+  def parse(html) do
```

Comment:
I'm about to change those interface modules to have a `parse_document/2` and `parse_fragment/2`, and remove the `parse/1`. Not sure when, but this should be made in order to fix the different output from parsers in test.

Reply:
Yeah I saw your comment. `fast_html` doesn't support parsing fragments now, but it's on my list to add right after this is merged.

Reply:
Added fragment support in https://git.pleroma.social/pleroma/fast_html/merge_requests/14. Want me to add new functions now?

Also what is the second argument in `parse_document/2` supposed to be? I assume it's the context tag for `parse_fragment/2`, but you don't need that when parsing documents?

Reply:
I don't think we should do this in this pull request. It will be awesome to fix this in another PR, but I think we need also to support fragments for html5ever, which is not implemented. I will take a look and check if it's OK to do that in html5ever_elixir.
But let's think better on those functions in another context. And thank you for adding that feature!